crmd/lrm.c

```diff
@@ -1779,6 +1860,7 @@ do_lrm_invoke(long long action,
             delete_resource(lrm_state, rsc->id, rsc, NULL, from_sys, from_host, user_name, input, unregister);
 
         } else {
+            record_pending_op(lrm_state, input->xml);
```

Comment:
Makes sense. We probably as well just use the already constructed op in there for record_pending_op().